lib/sparse/Cargo.toml

```diff
@@ -14,6 +14,7 @@ common = { path = "../common/common" }
 io = { path = "../common/io" }
 memory = { path = "../common/memory" }
 memmap2 = "0.9.0"
+schemars = { version = "0.8.15", features = ["uuid1", "preserve_order", "chrono"] }
```

Comment:
do we need all those features? uuid1 and chrono do not seem applicable I'd say

Reply:
Fixed